shopinvader_assortment/models/shopinvader_backend.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ShopinvaderBackend(models.Model):
+
+    _inherit = 'shopinvader.backend'
+
+    product_assortment_id = fields.Many2one(
+        comodel_name="ir.filters",
+        help="Bind only products matching with the assortment domain",
+        domain=[('is_assortment', '=', True)],
+        context={
+            'product_assortment': True,
+            'default_is_assortment': 1},
```

Comment:
`default_is_assortment`is computed by the default method in `product_assortment` IMO this default should be removed